app/views/videos/show.html.haml

```diff
@@ -2,14 +2,38 @@
   .container
     .row
       .video_large_cover.col-sm-7.col-sm-offset-1
-        %img(src="tmp/monk_large.jpg")
+        %img(src="#{@video.large_cover_url}")
```

Comment:
I would rather use `image_tag @video.large_cover_url` as its more succinct and readable.